### PR TITLE
expandOnHover attribute should consider numeric value

### PR DESCRIPTION
**Pull request by jacobscarter**
@StefanCodes  Here you go, I made a new PR from the same branch. 

With this use can set the time to expand the node.

### CODE REVIEW

**Comment by Voles** [comment]
@jacobscarter could you please remove the files from the `dist/` directory? Thanks! 👍

File: source/directives/uiTreeNode.js [hunk 1]
```diff
@@ -63,7 +63,7 @@
             });
 
             scope.$watch(attrs.expandOnHover, function(val) {
-              if ((typeof val) == 'boolean') {
+              if ((typeof val) == 'boolean' || (typeof val) == 'number') {
```
Comment (Voles) [inline source/directives/uiTreeNode.js]: @StefanCodes was this [the regression issue you've mentioned](https://github.com/angular-ui-tree/angular-ui-tree/pull/806#issuecomment-244822788)?

Comment (Voles) [inline source/directives/uiTreeNode.js]: @jacobscarter can you use `===` here? The return value of `typeof` is a string, so using the triple equals makes it clear that we're comparing strings here.

**Comment by jacobscarter** [comment]
@Voles I just re-opened this PR, it isn't my PR.  @pkatta can you address @Voles comments please

**Comment by Voles** [comment]
@jacobscarter I see now. Can you explain why this branch was originally created by @pkatta and why you've created a PR of someone else's branch?

I thought this PR was a fix for #806, but this commit is older than the original PR :confused:

**Comment by jacobscarter** [comment]
What happened was @StefanCodes merged @pkatta's PR, had to roll back master because of regressions caused by a merged PR, so @StefanCodes requested that this PR get re-opened so it can get re-merged.  See comments in this PR

https://github.com/angular-ui-tree/angular-ui-tree/pull/792

It was a bit of a chaotic situation where a PR was merged that caused issues so a rollback happened and PRs needed to get re-opened... When I re-submit my performance PR I will update unit and e2e tests so they are more thorough for the whole repo so we can avoid this in the future.

**Comment by Voles** [comment]
@jacobscarter thanks for making that clear for me. I've commented to @StefanCodes in #792

File: source/directives/uiTreeNode.js (hunk 1 above)
Comment (StefanCodes) [inline source/directives/uiTreeNode.js]: This wasn't the regression (that was in #806), but this PR was involved in the release.

**Comment by StefanCodes** [comment]
Fundamentally, I merged 2 PRs yesterday, released them, and 10 minutes later found a serious regression I'd missed. I resolved the problem upstream by resetting head and pushing, as re-committing an "undo" was extremely tedious due to a large number of commits (and merges).
I would have gotten away with it, too, if it hadn't been for GitHub! Unfortunately, rewriting history doesn't trigger Github to reset the merged status on any PRs that were nuked.

This PR and #792 are identical, and @pkatta is the original author of this change. If we re-merge #792 and delete this one, great. I think we should leave this one open until that happens, though, since #792 is still marked as closed and it risks falling out of sight.

**Comment by jacobscarter** [comment]
@StefanCodes to @Voles point, if @pkatta doesn't get back to us shortly lets make the code changes @Voles suggested ourselved and get this merged so we can get this confusion behind us :P

**Comment by Voles** [comment]
@StefanCodes @jacobscarter I've created #812 which contains the changes of this PR including my review feedback. In my opinion it would be good to merge that PR if you guys have no more feedback.

By doing so we should be back on track and ready to create a new release. Am I correct, @StefanCodes?

**Comment by StefanCodes** [comment]
@Voles correct! 2.18 would have included only #792 and #806. If we merge this then we only have #806 to look at, or we can start merging other PRs. Notably, #780 has been waiting for a while.

**Comment by jacobscarter** [comment]
@StefanCodes @Voles In relation to what I was saying earlier, I think we should merge this PR, send out a new release (2.18) and then continue merging other PRs.

Until we get things streamlined and more efficient I think it is a good idea to create a new release per PR so any rollbacks that may be needed don't cause a situation like this one.

**Pull request closed without merging**